### ISSUE TITLE
Error generating kallax.go: expected operand, got '%' (and 6 more errors)

### ISSUE THREAD

**Issue by ajnavarro** [src-d/go-kallax#69]
The complete message is:
```
kallax: No file generated due to an occurred error:
kallax.go:881:1: expected operand, found '%' (and 6 more errors)
```

The lines that throws the error are:
```
879 Timestamps:&schemaRepositoryReferencesTimestamps{
880 JSONSchemaKey: kallax.NewJSONSchemaKey(kallax.JSONAny),
881 %!(EXTRA string="_references", "Timestamps")CreatedAt:kallax.NewJSONSchemaKey(kallax.JSONAny, "_references", "Timestamps", "CreatedAt"),
882 UpdatedAt:kallax.NewJSONSchemaKey(kallax.JSONAny, "_references", "Timestamps", "UpdatedAt"),
883 },

```

Main model:
```
type Repository struct {
	kallax.Model `table:"repositories"`
	kallax.Timestamps `kallax:",inline"`
	Endpoints []string
	Status FetchStatus
	FetchedAt time.Time
	FetchErrorAt time.Time
	LastCommitAt time.Time
	References []*Reference
}
```

Reference model:
```
type Reference struct {
	kallax.Timestamps
	Name string
	Hash SHA1
	Init SHA1
	Roots []SHA1
}
```

**Comment by ajnavarro** [comment]
The bug apparently is in this line: https://github.com/src-d/go-kallax/blob/4a448f451c19cff86ca78eeea7abcd073bd15509/generator/template.go#L265

You are trying to print two elements and the string is only accepting one.

**Comment by erizocosmico** [comment]
Fixed in #73
